tests/lib/snaps/snapctl-from-snap/bin/app2

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+snapctl set foo=123
+snapctl get foo
```

Comment:
Instead of having several mini-apps with custom logic, we can have just two: snapctl-set and snapctl-set. The spread test can then define the logic being tested locally (set+get, etc).